pkg/kritis/crd/securitypolicy/securitypolicy_test.go

```diff
@@ -208,3 +208,92 @@ func Test_OnlyFixesNotAvailablePassWithWhitelist(t *testing.T) {
 		t.Errorf("got unexpected violations: %v", violations)
 	}
 }
+
+func Test_BuiltProjectIDs(t *testing.T) {
```

Comment:
NIT: How about using table testing? Current test cases are written in the table testing way (Not all though).

Reply:
Yes! I agree with you.
So fixed in f9d8b61.